core/standard/clause_6_overview.adoc

```diff
@@ -1,7 +1,14 @@
 [[overview]]
 == Overview
 
-NOTE: Need to have a disucssion in here about the migration of catalogues from big repositories of metadata documents (e.g. ISO19115) to distributed, crawlable catalogies (e.g. static STAC) and harvesting of resource into the catalogue to make them searchable (e.g. harvesting records close to resources (i.e. STAC item) or even harvesting the resource itself (e.g. the collection object)).
+[[records-role]]
+=== Role of OGC API Records
+
+Historically, catalogues have been used as big repositories of formal metadata like ISO-19119, ISO-19115, FGDC. etc. However, more recently catalogues are migrating to a more distributed model where the metadata is intimately associated with the resource that it describes. A modern example is STAC which relies on well defined building blocks that can be linked in different ways.
+
+Althought OGC API - Records is built upon the legacy of CSW, its role in a modern SDI can be both: a big repository of metadata documents or a distributed (even crawlable) catalogue.
```

Comment:
Although

Reply:
Update to:

Historically, catalogues have been used as large repositories of formal metadata like ISO 19119, ISO 19115, FGDC, etc. However, more recently catalogues are migrating to a more distributed model where the metadata is intimately associated with the resource that it describes. A modern example is STAC which relies on well defined building blocks that can be linked in different ways.

Reply:
Fixed, thanks